### ISSUE TITLE
MAIL/Reportsend  to subcriber group by mail relay server

### ISSUE THREAD

**Issue by blacks7** [ExchangeAnalyzer/ExchangeAnalyzer#172]
Can u add in paramter to define mail relay server, from address and to address. So the output could daily send to Administrator Group.

.PARAMETER SendEmail
Sends the HTML report via email using the SMTP configuration within the script.

**Comment by cunninghamp** [comment]
Considering that a lot of items in this report would require planning and change management processes to be followed before they can be fixed, do you still think a daily report will be useful?

**Comment by blacks7** [comment]
yes, when u have an big enviroment with many Exchange Server and a Team who administrate it in different Country, i want to know, when wrong changes where made.

So I can see the Problem on next executation of the check script and not when an user informed me about a Problem.

**Comment by chrisbrownie** [comment]
Most of the changes Exchange Analyzer will tell you about should be handled by change management procedures in an organisation of that size. Have you considered using the built-in tools for this? Exchange has [Admin Audit Logging](https://technet.microsoft.com/en-us/library/dn342832(v=exchg.150).aspx) for this very purpose.

**Comment by cunninghamp** [comment]
Thinking more on this, ExAnalyzer was not intended to be used in that way (as a monitoring or configuration drift tool). I can see the value in sending an email with the report, so that's something we can take on board as a feature request.
